index.css

```diff
@@ -87,6 +96,216 @@ nav {
   }
 
   .social-icon:hover {
-    width: 25px;
+    color: #396df2;
+  }
+
+  .works-sec {
+    display: grid;
+    grid-template-rows: auto auto auto auto;
+  }
+
+  .work-card {
+    display: grid;
+    grid-template-rows: auto auto auto auto;
+    gap: 10px;
+    width: 327px;
+    height: 552px;
+    padding: 16px;
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+    align-items: center;
+    background: #fff;
+  }
```

Comment:
- I guess you may have implemented something that has introduced a bug. There is no space that clearly differentiates your `<section class="catch-text-sec">` from your `<section class="works-sec">`. Can we fix this?? 🤗 

- Can we also ensure that the images on each card are properly centered?? Right now, it is a little too left-aligned. 🌸 

Here is what i can see 👇🏽 

| No space on Your Design  | Figma Design |
| :---:   | :---: | 
| ![nspic](https://user-images.githubusercontent.com/99246359/205316694-f18bc518-8c43-4970-a78d-7796c837de47.PNG) |![spfig](https://user-images.githubusercontent.com/99246359/205316875-4ea4a905-4842-4588-aaba-8fb856d8fddf.PNG)  |


- Your projects cards are currently just stacked next to each other without proper spacing. Just as the previous reviewer suggested, the spacing(margin, padding) must be similar to the original template provided in Figma. Can we implement this? 

Here is what's being rendered currently on my screen 👇🏽 

![nosp](https://user-images.githubusercontent.com/99246359/205317793-2448621e-70f6-43b2-8300-5f280832032c.PNG)

Reply:
@Favourezeugwa thanks a lot. On that in a blink